resources/js/beatmap-discussions/beatmap-owner.tsx

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+import UserAvatar from 'components/user-avatar';
+import UserLink from 'components/user-link';
+import UserJson from 'interfaces/user-json';
+import * as React from 'react';
+import { classWithModifiers } from 'utils/css';
+
+interface Props {
+  editing: boolean;
+  onRemoveUser?: (user: UserJson) => void;
+  user: UserJson;
+}
+
+function createRemoveOwnerHandler(user: UserJson, onRemoveClick?: NonNullable<Props['onRemoveUser']>) {
```

Comment:
there's no need for this lint workaround anymore now that the component is split out - this can just be part of the component